packages/@react-spectrum-uxp/storybook/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "rs3uxp-plugin",
+  "version": "3.0.0-alpha.1",
+  "private": true,
+  "license": "MIT",
+  "main": "dist/index.html",
+  "scripts": {
+    "watch:parcel": "rm -rf ../../../.parcel-cache && parcel serve src/index.html --target main --no-content-hash --no-cache --log-level verbose",
+    "watch-:uxp": "sleep 20 && yarn uxp",
+    "watch": "run-p watch:*",
+    "uxp": "../../../../torq-native/build/out/bin/macos/v8/nativeui/Debug/demo.app/Contents/MacOS/demo --extension dist"
```

Comment:
this seems very dependent on what you've done to your local system, can we generalize this a bit, maybe allow the path to be passed in? or instructions to add it to the PATH?
also, it's only going to work for Mac developers, is that alright?

Reply:
Not that specific to my system, but it is mac specific. I will change that to uxp-mac and later we can make a uxp-win.  This just requires torq-native to be installed as a peer directory to the monorepo. The simplicity of the script also makes it immediately clear to anyone capable of installing torq-native that they need to run the demo app (wherever theirs is located) and they can make their own simple shell script as needed.

I will definitely documenting the full setup so that anyone at Adobe can in principle set this up, but I expect that not many will want to setup torq-native (UXP) from scratch just for this.

Longer term, my goal is to provide a much easier way for RS3 developers to test components against UXP... either via a XD plugin or a standalone executable... possibly a UXP Browser (which I've been pushing for).